bundles/org.openhab.binding.fmiweather/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fmiweather"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Things -->
+	<thing-type id="observation">
+		<label>FMI current weather (observation)</label>
+		<description>Finnish Meteorological Institute (FMI) weather observation</description>
+
+		<channel-groups>
+			<channel-group id="current" typeId="current-group" />
+		</channel-groups>
+
+		<config-description>
+			<parameter name="fmisid" type="text" required="true">
+				<label>FMISID of the weather station</label>
+				<description><![CDATA[Station ID (FMISID) of the weather observation station
+				<br />
+				<br />See https://en.ilmatieteenlaitos.fi/observation-stations for a list of observation stations]]></description>
```

Comment:
Worth to mention that only ids of 'weather' stations do make sense here?

Reply:
Clarified instructions in d917bdc. In principle one can pick e.g. precipitation observation station and only that data would show up.